packages/@okta/vuepress-site/docs/guides/build-sso-integration/create-your-app/openidconnect/create.md

```diff
@@ -3,6 +3,7 @@
 4. On the Create a new app integration page, select **OpenID Connect** in the **Sign-in method** section.
 1. Choose either **Web Application** or **Single-Page Application** as the **Application type** for your integration. The OIN only publishes cloud-based SaaS apps, either traditional Web applications with a back-end or a modern browser-based SPA.
 1. Click **Next**.
-1. On the **General Settings** tab, enter a name for your integration and (optionally) upload a logo.
-1. On the **Configure OpenID Connect** tab, add your **Sign-in redirect URIs** and optional **Sign-out redirect URIs**. These URIs must be [absolute URIs](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#URI_resolution), and you can specify more than one.
+1. In **General Settings**, enter a name for your integration and (optionally) upload a logo.
```

Comment:
Check with @susanharper-okta on not using the "tab" part - that's in fairly common use in the dev docs library and would mean changing a number of the guides to be consistent if that's what we want to do stylistically.

Reply:
**General Settings** isn't a tab but a section on the New Web App Integration page.